.github/workflows/build_for_nix.yml

```diff
@@ -0,0 +1,45 @@
+name: Build WasmEdge on Nix
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "flake.nix"
+      - "flake.lock"
+      - ".github/workflows/build_for_nix.yml"
+      - "include/**"
+      - "lib/**"
+      - "!test/**"
```

Comment:
Oh wait, if just remove the 4 lines with `test/**` if you don't want to watch on tests at all lol